src/Equinox.SqlStreamStore/SqlStreamStore.fs

```diff
@@ -405,8 +405,10 @@ type Context(conn : Connection, batching : BatchingPolicy) =
 
 [<NoComparison; NoEquality; RequireQualifiedAccess>]
 type AccessStrategy<'event,'state> =
-    | EventsAreState
-    | RollingSnapshots of isValid: ('event -> bool) * compact: ('state -> 'event)
+    | LatestKnownEvent
+    /// Ensures a snapshot is always present (embedded in the stream as an event) every `batchSize` events using
```

Comment:
Anything else to call out in this comment? @bartelink

Reply:
how about including the link to the ES article?
also should clone the comment for LatestKnownEvent

Reply:
For the `LastestKnownEvent` comment:

`Treat every known event type in the Union as being an Origin event` -> `Load latest event type which is defined in the events union`?

Reply:
(added a separate comment - feel free to improve on it. I was reluctant to use Union as it doesn't strictly have to be a Union, i.e. if not using the FsCodec.NewtonsoftJson.Codec but I do accept its clearer to just talk about the common case)